planex/repository.py

```diff
@@ -143,13 +143,16 @@ def parse_bitbucket(self):
             query = query_str[3:]
             if '&' in query:
                 query = query[:query.find('&')]
-            query_path = query.split('/')
```

Comment:
I think it would be better to use urlparse.parse_qs(query_str), which splits the query string into a dict.